src/components/Dashboard.tsx

```diff
@@ -1,53 +1,22 @@
+import { useCallback, useEffect, useState } from 'react'
```

Comment:
My editor automatically sorts imports. Let me know if I should remove those sorting changes.

Reply:
It's fine. If anything, we should probably add our own config for a more consistent behaviour. We can just leave it like this for now.